sources/packages/api/src/route-controllers/student-appeal/student-appeal.students.controller.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  Controller,
+  Param,
+  Post,
+  Body,
+  NotFoundException,
+  UnprocessableEntityException,
+} from "@nestjs/common";
+import {
+  StudentAppealService,
+  ApplicationService,
+  FormService,
+} from "../../services";
+import { StudentAppealDTO } from "./models/student-appeal.dto";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { AllowAuthorizedParty, UserToken } from "../../auth/decorators";
+import { IUserToken } from "../../auth/userToken.interface";
+import {
+  ApiTags,
+  ApiCreatedResponse,
+  ApiNotFoundResponse,
+  ApiUnprocessableEntityResponse,
+} from "@nestjs/swagger";
+import BaseController from "../BaseController";
+import { ClientTypeBaseRoute, ApiProcessError } from "../../types";
+import { INVALID_APPLICATION_NUMBER } from "../../constants";
+
+@AllowAuthorizedParty(AuthorizedParties.student)
+@Controller("appeal")
+@ApiTags(`${ClientTypeBaseRoute.Student}-appeal`)
+export class StudentAppealStudentsController extends BaseController {
+  constructor(
+    private readonly studentAppealService: StudentAppealService,
+    private readonly applicationService: ApplicationService,
+    private readonly formService: FormService,
+  ) {
+    super();
+  }
+
+  /**
+   * Submit student appeal.
+   * @param applicationId application for which the appeal is submitted.
+   * @param payload student appeal with appeal requests.
+   * @param userToken
+   */
+  @ApiCreatedResponse({ description: "Student appeal created successfully" })
+  @ApiNotFoundResponse({
+    description:
+      "Application either not found or not eligible to request for change.",
```

Comment:
Can we change to call it an appeal?
`Application either not found or not eligible to request an appeal.`